sample/buildSrc/build.gradle

```diff
@@ -0,0 +1,21 @@
+repositories {
+    jcenter()
+}
+
+apply plugin: 'groovy'
+
+dependencies {
+    compile gradleApi()
+}
+
+sourceSets {
```

Comment:
I could not load both plugin and sample in IDEA before. This approach links buildSrc explicitly.